crates/holochain_zome_types/src/call_remote.rs

```diff
@@ -19,34 +19,34 @@ impl CallRemote {
         zome_name: ZomeName,
         fn_name: FunctionName,
         cap: Option<CapSecret>,
-        request: SerializedBytes,
+        payload: ExternIO,
     ) -> Self {
         Self {
             to_agent,
             zome_name,
             fn_name,
             cap,
-            request,
+            payload,
         }
     }
 
-    pub fn to_agent(&self) -> AgentPubKey {
-        self.to_agent.clone()
+    pub fn as_to_agent(&self) -> &AgentPubKey {
```

Comment:
Why rename these at all? It's pretty idiomatic to have accessors with the same name as the underlying field. Is it just because `to_agent()` no longer sounds like an accessor? If so, I would suggest renaming `to_agent` to not sound like a function. Like `target_agent` maybe?

Reply:
https://rust-lang.github.io/api-guidelines/naming.html because this is a newly borrowed to borrowed conversion that is free, much like `AsRef`

Reply:
i did a combination of making the functions `foo_as_ref` as per @steveeJ and renamed `to_agent` to `target_agent` as per @maackle and kept the `as` vibe as per naming conventions for borrowed return values